packages/components/src/Enumeration/Header/HeaderSelected.component.js

```diff
@@ -17,7 +17,9 @@ function getAction(action, index) {
 			action.onClick(event);
 		}
 	}
-
+	if (action.disabled) {
+		return null;
+	}
```

Comment:
Same remark : You should filter the actions in the main component return, before the map()